### DISCUSSION TITLE
Website + Blog + Forum Built with Nuxt + Strapi + GraphQL

### DISCUSSION THREAD

**Discussion by sandiprb** [strapi/strapi#8000] [Legacy Topics]
Website: https://www.beshak.org
Blog: https://www.beshak.org/articles
Forum: https://www.beshak.org/forum

Stack: Nuxt JS + Strapi + GraphQL

I have also open sourced the forum module, available here: https://github.com/sandiprb/strapi-forum

**Comment by derrickmehaffy** [comment]
Very nice @sandiprb ! 
@yvesdo  @malgamves

**Reply by yvesdo** [reply]
Super neat! 

@sandiprb, would you be available for an interview with @shpiliak-a and me? The idea would be to understand your needs when you chose Strapi and to have an in-depth understanding of how you used Strapi to create the different functionalities.
If you agree, we could then showcase you in a case study on our website? 

You can schedule a meeting with me [here](https://meetings.hubspot.com/yves20)!

**Reply by sandiprb** [reply]
Done. Scheduled a call @yvesdo 
Would love to chat  with you guys.
